project-api/project-api-test/src/test/resources/myproject_with_lang_import/modules/services/resources/invalidProject/main.bal

```diff
@@ -0,0 +1,5 @@
+import ballerina/io;
```

Comment:
I think we can't use imports from central.

Reply:
This resource project is a modified one which is already available in the repo ("myproject"). Any suggestions?